components/massmint/uploadCompressedMedia/useZipValidator.ts

```diff
@@ -39,6 +39,27 @@ export const validFormats = [
   'json',
 ]
 
+const mimeTypes: { [key: string]: string } = {
```

Comment:
maybe move it to static so we can reuse it in new minting process

Reply:
I did, and i'm not sure why but
``` import { mimeTypes } from '@kodadot1/static' is failing```
so for now i've kept it in ```useZipValidator``` as well

Reply:
> I would imo move it to separate package :D


![simpson-homer-simpson](https://github.com/kodadot/nft-gallery/assets/9987732/3a1b0d3d-4556-43e1-8b0b-106e99eb269d)